platform/admin/ui/pom.xml

```diff
@@ -66,6 +68,10 @@
             <type>tar.gz</type>
             <classifier>npm</classifier>
         </dependency>
+        <dependency>
```

Comment:
With this added here, do you still need the `org.apache.karaf.webconsole.console` dependency in this pom?